queries.sql

```diff
@@ -145,3 +145,85 @@ JOIN OWNERS o ON a.OWNERS_ID = o.ID
 GROUP BY o.full_name
 ORDER BY COUNT(a.name) DESC;
 
+-- DAY 4
+
+-- Who was the last animal seen by William Tatcher?
+
+SELECT A.NAME,VE.NAME AS VETS_NAME , MAX(DATE_VISITED) AS LAST_VISIT
+FROM ANIMALS A
+JOIN VISITS VI ON A.NAME = VI.ANIMAL_NAME
+JOIN VETS VE ON VI.VETS_NAME = VE.NAME
+WHERE VE.NAME = 'William Tatcher'
+GROUP BY A.NAME,VE.NAME
+ORDER BY MAX(DATE_VISITED) DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT VE.NAME,COUNT(DISTINCT VI.ANIMAL_NAME) FROM
+VISITS VI
+JOIN VETS VE ON VI.VETS_NAME = VE.NAME
+WHERE VE.NAME = 'Stephanie Mendez'
+GROUP BY VE.NAME
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT *
+FROM VETS VE
+FULL JOIN SPECIALIZATIONS SP
+FULL JOIN SPECIES S ON SP.SPECIES_NAME = S.NAME
+ON VE.NAME = SP.VET_NAME
+
+--List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+
+SELECT VE.NAME AS VET,A.NAME AS ANIMAL,VI.DATE_VISITED
+FROM ANIMALS A
+JOIN VISITS VI ON VI.ANIMAL_NAME = A.NAME
+JOIN VETS VE ON VE.NAME = VI.VETS_NAME
+WHERE VE.NAME = 'Stephanie Mendez'
+AND VI.DATE_VISITED = '2020-AUG-30';
+
```

Comment:
- [OPITONAL] Note here, the date should be between April 1st and August 30th, 2020 💡